keyvi/tests/keyvi/util/vint_test.cpp

```diff
@@ -58,16 +58,33 @@ BOOST_AUTO_TEST_CASE(VShortLength) {
   BOOST_CHECK_EQUAL(77777, decodeVarshort(buffer));
 
   encodeVarshort(32767, buffer, &size);
-  BOOST_CHECK_EQUAL(util::getVarshortLength(1), size);
```

Comment:
Note: a unit test was in place but the check was rubbish. This should have been:

```
BOOST_CHECK_EQUAL(util::getVarshortLength(32767), size);
```

or 

```
BOOST_CHECK_EQUAL(1, size);
```